sunpy/io/special/srs.py

```diff
@@ -161,6 +160,8 @@ def get_meta_data(header):
 
     meta_data = {}
     for m in meta_lines:
+        if "Corrected Copy" in m:
+            continue
```

Comment:
Not sure we need it saying corrected in the header? 

I can work around it if we really want it.